### PR TITLE
Feature 995

### PR DESCRIPTION
**Pull request by MashB**
Task
https://dev.azure.com/TDEI-UW/TDEI/_workitems/edit/995/

### CODE REVIEW

File: tdei-api-gateway-stage.json [hunk 1]
```diff
@@ -2181,24 +2562,274 @@
...
+                                "description": "Location inaccuracy factors"
+                            }
+                        }
+                    },
+                    "dataset_detail": {
+                        "type": "object",
+                        "properties": {
+                            "name": {
+                                "type": "string",
+                                "description": "Dataset name or Title that this data is known by"
```
Comment (susrisha) [inline tdei-api-gateway-stage.json:2606]: please keep as `title`

File: tdei-api-gateway-stage.json [hunk 2]
```diff
@@ -2181,24 +2562,274 @@
...
+                                "type": "string",
+                                "description": "Dataset version"
+                            },
+                            "custom_metadata": {
+                                "type": "object",
+                                "description": "Custom structured JSON metadata"
+                            },
+                            "collected_by": {
+                                "type": "string",
+                                "description": "Description of who data was collected by. See Best Practices document for information on how to format this string.",
```
Comment (susrisha) [inline tdei-api-gateway-stage.json:2622]: do we have a best practices document?

File: tdei-api-gateway-stage.json [hunk 3]
```diff
@@ -2181,24 +2562,274 @@
...
+                                "example": "See best practices document"
+                            },
+                            "collection_date": {
+                                "type": "string",
+                                "description": "date-time that data was collected",
+                                "example": "2018-02-10T09:30Z"
```
Comment (susrisha) [inline tdei-api-gateway-stage.json:2628]: Please keep a better example. We always got confused whether 02 is month or day? A better example would be 2019-22-10

**Review by susrisha — CHANGES_REQUESTED** [review]
minor edits to the descriptions.

File: tdei-api-gateway-stage.json (hunk 2 above)
Comment (MashB) [reply tdei-api-gateway-stage.json:2622]: Removed , we do not have.

File: tdei-api-gateway-stage.json (hunk 1 above)
Comment (MashB) [reply tdei-api-gateway-stage.json]: updated

**Review by susrisha — APPROVED** [review]

**Pull request merged**